Bottom: 9cea3990b723d805583c621d3b0ae1a1f7d4263c
Top:    6de3e83943a23276cab8af1e6e168e504b58c76f
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-12 12:14:33 +0800

autofs-5.0.7 - don't override LDFLAGS in make rules

Ensure that externally defined LDFLAGS is not overridden.


---

diff --git a/Makefile.rules b/Makefile.rules
index f2ba386..6b5b2bd 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -24,16 +24,16 @@ endif
 
 ifdef DEBUG
 CFLAGS    ?= -g -Wall -DDEBUG
-LDFLAGS   = -g
+LDFLAGS   ?= -g
 STRIP     = :
 else
 ifdef DONTSTRIP
 CFLAGS    ?= -O2 -g
-LDFLAGS   = -g
+LDFLAGS   ?= -g
 STRIP     = :
 else
 CFLAGS    ?= -O2 -Wall
-LDFLAGS   = -s
+LDFLAGS   ?= -s
 STRIP     = strip --strip-debug
 endif
 endif
